mllib/src/main/scala/org/apache/spark/ml/util/Instrumentation.scala

```diff
@@ -253,6 +254,13 @@ private[spark] class OptionalInstrumentation private(
     }
   }
 
+  override def logInfo(logEntry: LogEntry): Unit = {
```

Comment:
why do we need this?

Reply:
This is so that classes that use the `OptionalInstrumentation` for logging such as [WeightedLeastSquares](mllib/src/main/scala/org/apache/spark/ml/optim/WeightedLeastSquares.scala), [CrossValidator](mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala) can also log MessageWithContext. Else I will get `Type Mismatched String => MessageWithContext`